packages/x-components/src/components/base-grid.vue

```diff
@@ -37,6 +37,18 @@
   import { XInject } from './decorators/injection.decorators';
   import { LIST_ITEMS_KEY } from './decorators/injection.consts';
 
+  /**
+   * The type returned by the gridItems function. Basically it's a list of items with its CSS
+   * classes and a slotName.
+   *
+   * @public
```

Comment:
When not exported, the public/internal/private decorators are worthless, they won't be used by API extractor. However, I think it still is a good practise to explicitly say this way if we intend something to be only for internal developers. So either remove this `public`, or change it to `internal`, whatever you prever.

Reply:
My fault, it's internal.